ci-operator/config/openshift/loki/OWNERS

```diff
@@ -9,6 +9,7 @@ approvers:
 - jcantrill
 - periklis
 - xperimental
+- cahartma
```

Comment:
You are not allowed to change this file, this is sync'ed from the original repo

Reply:
reverted